src/Model/AvailabilityItem.php

```diff
@@ -25,33 +25,35 @@ class AvailabilityItem extends Entity
 {
 
     /**
-    * Gets the endTime
+    * Gets the endDateTime
+    * The end time of the time slot.
     *
-    * @return TimeOfDay|null The endTime
+    * @return DateTimeTimeZone|null The endDateTime
     */
-    public function getEndTime()
+    public function getEndDateTime()
```

Comment:
Confirmed with Vincent that this won't be treated as a breaking change.
Previous version of the metadata didn't last more than a week in v1 & the workload was still returning `DateTimeTimeZone` anyway despite metadata showing `TimeOfDay`